config/test/affine.yaml

```diff
@@ -18,6 +18,7 @@ train:
   preprocess:
     batch_size: 2
     shuffle_buffer_num_batch: 1
+    num_parallel_calls: -1 # number of workers to be used, -1 means unlimited
```

Comment:
"workers" is a bit vague... Can we clarify what this actually does for the layperson?

Reply:
I wish I know exactly what it is.

Here's tf doc
- https://www.tensorflow.org/api_docs/python/tf/data/Dataset#map
- https://www.tensorflow.org/guide/data_performance#parallelizing_data_transformation

I will try to clarify it a bit further. If still not ok, feel free to propose one or modify it on the branch directly.